### DISCUSSION TITLE
How can I get the node content at specific position

### DISCUSSION THREAD

**Discussion by guoz2013** [singerdmx/flutter-quill#894] [Feature Request]
When I click on the editor, how do I get the specific node content of the click location, I know I can queryNote by offset, but what I get is a Line, and it might have multiple nodes, but I want a specific node, 
Thank you

**Comment by singerdmx** [comment]
<img width="1165" alt="image" src="https://user-images.githubusercontent.com/122956/181791644-603b713d-19a4-415d-99a0-ad4c814276c0.png">

**Reply by guoz2013** [reply]
I tried this method, but  it causes Stack Overflow when executed, could this be a bug ?

QuillEditor(
              controller: qc,
              readOnly: false,
              focusNode: FocusNode(),
              showCursor: true,
              scrollController: ScrollController(),
              scrollable: true,
              padding: const EdgeInsets.all(10),
              autoFocus: true,
              expands: false,
              placeholder: "",
              minHeight: 60,
              onTapUp: (detail, fn) {
                int gestureOffset = fn(detail.localPosition).offset;
                var leafNode = qc.document.querySegmentLeafNode(gestureOffset);
                print(leafNode);
                return false;
              },
            )

<img width="1467" alt="image" src="https://user-images.githubusercontent.com/38322204/181861806-5a2c3517-26a7-4941-b945-c1764eedfa38.png">

**Reply by singerdmx** [reply]
You need to debug it yourself. This is all the help I can do.
